xscen/regridding.py

```diff
@@ -42,6 +45,22 @@ def regrid(
       Supports a 'mask' variable compatible with ESMF standards.
     regridder_kwargs : dict
       Arguments to send xe.Regridder().
+    intermediate_reg_grids: dict
+        This argument is used to do a regridding in many steps, regridding to regular
+        grids before regridding to the final ds_grid.
+        This is useful when there is a large jump in resolution between ds and ds grid.
```

Comment:
This might be too much text for the arguments section of the Docstrings? We could move some of it to a "Notes" section (as `extract_dataset`, for example).

Reply:
template moved in notes!